src/nni_manager/training_service/kubernetes/frameworkcontroller/frameworkcontrollerApiClient.ts

```diff
@@ -20,35 +20,38 @@
 'use strict';
 
 import * as fs from 'fs';
-import { KubernetesCRDClient, GeneralK8sClient } from '../kubernetesApiClient';
+import { GeneralK8sClient, KubernetesCRDClient } from '../kubernetesApiClient';
 
-abstract class FrameworkControllerClient extends KubernetesCRDClient{
+// tslint:disable:completed-docs function-name no-use-before-declare
```

Comment:
For most cases, suggest not to make tslint error/warning disappear by "tslint:disable",  if some warnings/errors can not be fixed in this PR, just leave it there,  and maybe fixed in next PR.